README.md

```diff
@@ -35,9 +35,18 @@ GOPATH=`pwd`/Godeps/_workspace go build -o etcdenv .
 | ------ | ------- | ----------- |
 | `server`, `s` | http://127.0.0.1:4001 | Location of the etcd server |
 | `namespace`, `n`| /environments/production | Etcd directory where the environment variables are fetched. You can watch multiple namespaces by using a comma-separated list (/environments/production,/environments/global) |
-| `r` | false | Not restart the command when a key watch change |
+| `shutdownBehavour`, `b` | keepalive | Strategy to apply when the process exit, further information into the next paragraph |
```

Comment:
camelCase in command options looks weird to me ... is not `shutdown-behavior` nicer ? (plus the typo — Behavour)